netflix-roulette/src/components/AppHome/components/MoviesList/MoviesList.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { List, ListItem } from "@material-ui/core";
+import { Movie } from "../../AppHome";
+
+interface MoviesListProps {
+  movies: Movie[];
+}
+
+export class MoviesList extends React.Component<MoviesListProps, {}> {
+  render() {
+    return (
+      <List component="ul">
+        {this.props.movies.map((movie) => (
+          <ListItem>{movie.title}</ListItem>
```

Comment:
you should provide a key attribute for list items

Reply:
Good point, updated